src/Model/Feature.php

```diff
@@ -12,22 +12,22 @@
 interface Feature
 {
     /**
-     * Get key
+     * Get name.
      */
-    public function getKey(): string;
+    public function getName(): string;
```

Comment:
Not sure about changing the naming here.

This method refers to an identifier for the feature. `name` is too ambiguous IMO